### DISCUSSION TITLE
Can't upload proof of my teacher status and time is running out

### DISCUSSION THREAD

**Discussion by ignaciogarcia** [community/community#87494] [GitHub Education]
### Hi everyone,

Hello.

I've been using GitHub Classroom as University teacher for several years now (at University of Alcalá, Spain). Recently I received an e-mail notifying that my status expires on January 28th and that I needed to provide qualifying proof of current academic status.

However, I can't upload a picture of my University card. I select it on my hard drive but when I click on "Process my application" nothing happens. It says it might take a few seconds, but it just doesn't work.

Can someone please help me? My classes start next week and I'm running out of time.

Regards,

J. Ignacio García

**Comment by afeuerstein** [comment]
Seems to be the same problem as here: https://github.com/orgs/community/discussions/88401#discussioncomment-8148320
(#88401)

**Reply by ignaciogarcia** [reply]
Thank you, I resorted to taking a new picture using the webpage and that seemed to allow me to progress (still waiting for confirmation). We'll see. 

What is really terrible, though, is that GitHub forces you to resort to forums to solve this kind of stuff, instead of having some sort of support e-mail.

**Comment by rizmyabdulla** [comment]
Try this : https://github.com/orgs/community/discussions/88401#discussioncomment-8142514

**Reply by ignaciogarcia** [reply]
Thanks, I'll look into those solutions. For the time being, I took a new picture using the webpage and that seemed to work. We'll see...

Sad that one has to resort to forums for this, though...
